OTPKit/Features/TripPlanner/Direction/DireactionLegOriginDestinationView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  DireactionLegOriginDestinationView.swift
+//  OTPKit
+//
+//  Created by Hilmy Veradin on 08/08/24.
+//
+
+import SwiftUI
+
+struct DireactionLegOriginDestinationView: View {
```

Comment:
please rename to `DirectionLegOriginDestinationView` - you have an extra `a` in the first word :)

Reply:
Whoops. My bad. Thanks for pointing that out ^^ @aaronbrethorst